e2e/workspace/MODULE.bazel

```diff
@@ -4,12 +4,22 @@ local_path_override(
     path = "../..",
 )
 
+cypress = use_extension("@aspect_rules_cypress//cypress:extensions.bzl", "cypress")
+cypress.toolchain(cypress_version = "13.6.6")
+use_repo(cypress, "cypress_toolchains")
+
+register_toolchains("@cypress_toolchains//:all")
+
 bazel_dep(name = "aspect_rules_js", version = "1.41.2", dev_dependency = True)
 bazel_dep(name = "bazel_features", version = "1.9.0", dev_dependency = True)
 
 npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm", dev_dependency = True)
 npm.npm_translate_lock(
     name = "npm",
+    # Speed up installation by disabling cypress binary install. Optional.
+    lifecycle_hooks_envs = {
```

Comment:
Lets pre-factor all of the things in this PR that are not related to the "no-sandbox" change :)

This one, the prettier changes and "short" tags, can just go in a "cleanup" PR.